[06/29/18, 19:30:01:490] info: ╔═════════════════════════════════════════════════════════════════════════════════╗
[06/29/18, 19:30:01:491] info: ║      Slack 3.2.1, linux 4.10.0-38-generic on x64 at 06/29/18, 19:30:01:487      ║
[06/29/18, 19:30:01:491] info: ╚═════════════════════════════════════════════════════════════════════════════════╝
[06/29/18, 19:30:01:668] info: Initializing local storage instance at path: /home/nozomi/dotfiles/.config/Slack/local-settings.json
[06/29/18, 19:30:01:835] info: NetworkStatus: Waiting for network online…
[06/29/18, 19:30:02:140] info: Don't start background client because T031FNTUT is most used
[06/29/18, 19:30:02:147] info: Store: BACKGROUND_CLIENTS_INITIALIZED
[06/29/18, 19:30:02:154] info: Don't start background client because TB47MRNBZ is selected team
[06/29/18, 19:30:13:119] warn: SlackApi: Failed to call api.features
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:30:13:127] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4002,
  "teamId": "TB9PVC95H"
}
[06/29/18, 19:30:13:138] info: Store: BACKGROUND_CLIENTS_INITIALIZED true
[06/29/18, 19:30:17:609] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 6,
  "state": "loaded",
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:30:17:623] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 6,
  "state": "empty-page",
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:30:17:633] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 6,
  "state": "page-will-navigate",
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:30:17:640] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 6,
  "state": "page-will-navigate",
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:30:17:651] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 6,
  "state": "empty-page",
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:30:18:146] info: Store: START_BACKGROUND_CLIENT
{
  "teamId": "TB9PVC95H",
  "token": "xoxs-⋯"
}
[06/29/18, 19:30:28:306] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:30:28:306] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:30:28:308] warn: Tried to determine signed in status, but errored.
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:30:28:308] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:30:28:310] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:30:28:320] info: Store: UPDATE_SESSION_STATUS
{
  "hasValidSession": null,
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:30:28:711] info: Store: SET_NETWORK_STATUS offline
[06/29/18, 19:30:38:200] error: startBackgroundClientEpic: Socket error 4002
{
  "message": "The socket encountered a network error, reconnect when network online"
}
[06/29/18, 19:30:38:202] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4002,
  "teamId": "TB9PVC95H"
}
[06/29/18, 19:30:38:626] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 6,
  "state": "loaded",
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:30:38:637] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 6,
  "state": "empty-page",
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:30:38:641] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 6,
  "state": "page-will-navigate",
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:30:43:208] info: Store: START_BACKGROUND_CLIENT
{
  "teamId": "TB9PVC95H",
  "token": "xoxs-⋯"
}
[06/29/18, 19:30:48:317] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:30:48:318] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:30:48:319] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:30:48:320] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:30:48:341] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:30:48:342] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:30:48:352] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:30:48:352] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:03:119] warn: Tried to determine signed in status, but errored.
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:03:121] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:03:128] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:03:136] info: Store: UPDATE_SESSION_STATUS
{
  "hasValidSession": null,
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:31:08:235] error: startBackgroundClientEpic: Socket error 4002
{
  "message": "The socket encountered a network error, reconnect when network online"
}
[06/29/18, 19:31:08:243] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4002,
  "teamId": "TB9PVC95H"
}
[06/29/18, 19:31:08:336] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:08:336] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:08:337] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:08:337] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:08:337] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:08:337] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:13:262] info: Store: START_BACKGROUND_CLIENT
{
  "teamId": "TB9PVC95H",
  "token": "xoxs-⋯"
}
[06/29/18, 19:31:28:354] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:28:355] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:28:356] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:28:357] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:28:358] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:28:358] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:28:359] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:28:359] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:33:295] error: startBackgroundClientEpic: Socket error 4002
{
  "message": "The socket encountered a network error, reconnect when network online"
}
[06/29/18, 19:31:33:302] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4002,
  "teamId": "TB9PVC95H"
}
[06/29/18, 19:31:38:317] info: Store: START_BACKGROUND_CLIENT
{
  "teamId": "TB9PVC95H",
  "token": "xoxs-⋯"
}
[06/29/18, 19:31:40:127] info: Store: RELOAD
{
  "everything": false
}
[06/29/18, 19:31:40:138] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:40:139] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:40:139] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:40:139] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:40:140] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:40:141] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:40:142] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:40:143] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:40:143] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:40:144] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:40:144] info: NetworkStatus: Network check failed
{
  "error": {
    "message": "Failed to fetch",
    "stack": "TypeError: Failed to fetch"
  }
}
[06/29/18, 19:31:40:144] info: NetworkStatus: Response from api.test
{
  "status": "offline"
}
[06/29/18, 19:31:40:158] error: startBackgroundClientEpic: Socket error 4002
{
  "message": "The socket encountered a network error, reconnect when network online"
}
[06/29/18, 19:31:40:163] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 6,
  "state": "empty-page",
  "teamId": "TB47MRNBZ"
}
[06/29/18, 19:31:40:163] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4002,
  "teamId": "TB9PVC95H"
}
